modules/system/config-wfrp4e.js

```diff
@@ -1362,17 +1361,17 @@ WFRP4E.conditionScripts = {
             bleedingRoll = new Roll("1d100").roll().total;
             if (bleedingRoll <= bleedingAmt * 10)
             {
-                msg += `<br><b>${actor.data.token.name}</b> dies from blood loss! (Rolled ${bleedingRoll})`
+                msg += `<br>${game.i18n.localize("BleedUnc1")}<b>${actor.data.token.name}</b> ${game.i18n.localize("BleedFail")} (${game.i18n.localize("Rolled")} ${bleedingRoll})`
                 actor.addCondition("dead")
             }
             else if (bleedingRoll % 11 == 0)
             {
-                msg += `<br><b>${actor.data.token.name}'s</b> blood clots a little and loses 1 Bleeding Condition (Rolled ${bleedingRoll})`
+                msg += `<br><b>${actor.data.token.name}${game.i18n.localize("'s")}</b> ${game.i18n.localize("BleedCrit")} (${game.i18n.localize("Rolled")} ${bleedingRoll})`
```

Comment:
Should use `format` function, localizing `'s` is definitely not going to work for most languages.